src/tests/blueprint/t_blueprint_mpi_mesh_transform.cpp

```diff
@@ -364,6 +257,11 @@ TEST(conduit_blueprint_mesh_transform, amr_2d_transform_parallel)
     conduit::blueprint::mpi::mesh::to_polygonal(mesh, poly, "topo");
 
     EXPECT_TRUE( conduit::blueprint::mpi::verify("mesh",poly,info, MPI_COMM_WORLD));
```

Comment:
I think it would be good if there were a few more tests here to ensure that the new mesh has the expected structure, e.g.:

```
EXPECT_TRUE(conduit::blueprint::mesh::topology::unstructured::verify(poly["path/to/topo"]));
EXPECT_EQ(poly["path/to/topo/type"].as_string(), "polygonal");
```

Reply:
Also, some broad-level checks to ensure that the contents look correct:

```
EXPECT_EQ(/*# elements in polygonal topology*/, /*# elements in original structured AMR topology*/);
```